### PR TITLE
Add Camunda Modeler v3.4.0 release blog post

### PR DESCRIPTION
**Pull request by oguzeroglu**
Related to https://github.com/camunda/camunda-modeler/issues/1533

### CODE REVIEW

File: content/post/2019/10/camunda-modeler-3.4.0-released.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+A good example for a UI Plugin is our built-in Deployment feature.
+
+{{< figure class="no-border teaser" src="deployment_tool_as_a_plugin.gif" alt="Deployment Tool" caption="Built-in Deployment Tool as a UI plugin." >}}
+
+## Improved Deployment Experience
+
+Camunda Modeler v3.4 comes with an improved deployment experience for its users. The built-in Deployment Plugin will now automatically check if the engine backend is accessible or not before the deployment process. In addition, the Deployment Plugin is also now capable of remembering the deployment details per diagram. We believe this will make things much easier for our users as the deployment feature is meant to be used repeatedly.
+
+{{< figure class="no-border teaser" src="deployment.gif" alt="Deployment Tool" caption="Deployment Tool automatically checks if the server is accessible." >}}
```
Comment (pinussilvestrus) [inline content/post/2019/10/camunda-modeler-3.4.0-released.md]: Wouldn't say 'server', but 'Camunda Engine'

File: content/post/2019/10/camunda-modeler-3.4.0-released.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+We are happy to announce the release of Camunda Modeler v3.4. It is shipped with two
+important improvements: Extension point for UI Plugins and features for a better
+deployment experience. In addition, we included several bug fixes.
+
+[Download the latest release](https://camunda.com/download/modeler/) and start modeling.
+
+<!--more-->
+
+## Extension Point for UI Plugins
```
Comment (pinussilvestrus) [inline content/post/2019/10/camunda-modeler-3.4.0-released.md]: Personally saying I find the deployment improvements more important having in the first position.

Generally about this section: I like how the article describes the benefits for the actual User. Maybe we can add a link to the actual [showcases](https://github.com/camunda/camunda-modeler/tree/develop/resources/plugins) for examples on how to create the new client extension?

File: content/post/2019/10/camunda-modeler-3.4.0-released.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+## Improved Deployment Experience
+
+Camunda Modeler v3.4 comes with an improved deployment experience for its users. The built-in Deployment Plugin will now automatically check if the engine backend is accessible or not before the deployment process. In addition, the Deployment Plugin is also now capable of remembering the deployment details per diagram. We believe this will make things much easier for our users as the deployment feature is meant to be used repeatedly.
+
+{{< figure class="no-border teaser" src="deployment.gif" alt="Deployment Tool" caption="Deployment Tool automatically checks if the server is accessible." >}}
+
+## Looking Forward
+
+Read the [changelog](https://github.com/camunda/camunda-modeler/blob/master/CHANGELOG.md#340) to learn about all features and bug fixes that made it into this release. [Download the new modeler](https://camunda.com download/modeler/), give it a try, and, most importantly, give us feedback.
```
Comment (nikku) [inline content/post/2019/10/camunda-modeler-3.4.0-released.md]: Maybe mention that numerous additional improvements made it into the release.

File: content/post/2019/10/camunda-modeler-3.4.0-released.md (hunk 2 above)
Comment (nikku) [reply content/post/2019/10/camunda-modeler-3.4.0-released.md]: If we publicly marked that new plug-in capability, please add the fact that the new plug-in point is, like all our existing extension points APIs, not a stable API. Cf. our [plug-in documentation](https://github.com/camunda/camunda-modeler/tree/develop/docs/plugins).

**Comment by oguzeroglu** [comment]
I updated the blog post regarding your feedback.

**Pull request merged**